terragrunt/env/production/env_vars.hcl

```diff
@@ -1,9 +1,9 @@
 inputs = {
-    account_id        = "211125499457" 
-    env               = "production"
-    product_name      = "cra_upd_dashboard" 
-    cost_center_code  = "cra-upd-dashboard" 
-    billing_code      = "CostCentre" 
-    billing_tag_value = "CraUpdDashboard" 
-    domain            = "cra-arc.cdssandbox.xyz"
-  }
\ No newline at end of file
+  account_id        = "211125499457"
```

Comment:
This will be a production account, which we can create at a later time. You can leave it there, but just the heads up that it will change.